### PR TITLE
test(export): add export test

### PR DESCRIPTION
**Pull request by YardWill**
### 👀 PR includes

🔧 Chore

- [x] Test case
- [ ] Docs / demos update
- [ ] CI / workflow
- [ ] Release version
- [ ] Other ()

### 📝 Description

Add export test

### 🖼️ Screenshot

| Before | After |
| ------ | ----- |
| ❌      | ✅     |

### CODE REVIEW

File: packages/s2-core/__tests__/unit/utils/export/export-spec.ts [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+        },
+      }),
+      assembleOptions({ hierarchyType: 'grid' }),
+    );
+    s2.render();
+    const data = copyData(s2, '\t');
+    const rows = data.split('\n');
+    expect(rows.length).toEqual(14);
+    rows.forEach((e) => {
+      expect(e.split('\t').length).toEqual(34);
```
Comment (lijinke666) [inline packages/s2-core/__tests__/unit/utils/export/export-spec.ts:90]: 像这种判断长度, 可以直接用 jest 的语法糖 

```
expect(e.split('\t')).toHaveLength(34);
```

**Review by lijinke666 — APPROVED** [review]

**Review by xingwanying — COMMENTED** [review]
Pivot table还需要加上小计总计和序号的0，0

**Comment by YardWill** [comment]
> Pivot table还需要加上小计总计和序号的0，0

等会我再加一波

**Review by lijinke666 — APPROVED** [review]

**Review by xingwanying — APPROVED** [review]
LG

**Pull request merged**